From 13a8894fe2faa45b04033d7122a8fe7939ce6aa2 Mon Sep 17 00:00:00 2001
From: Gavin Smith <gavinsmith0123@gmail.com>
Date: Thu, 16 Sep 2021 19:06:28 +0100
Subject: * tp/Texinfo/XS/parsetexi/end_line.c (parse_line_command_args)
 <@headings>: Add missing option value 'single'.  Report from Sergei
 Trofimovich <slyich@gmail.com>.

---
 ChangeLog                          | 6 ++++++
 tp/Texinfo/XS/parsetexi/end_line.c | 4 ++--
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 8c1a0f3..5104f50 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,9 @@
+2021-09-16  Gavin Smith  <gavinsmith0123@gmail.com>
+
+	* tp/Texinfo/XS/parsetexi/end_line.c (parse_line_command_args)
+	<@headings>: Add missing option value 'single'.  Report from
+	Sergei Trofimovich <slyich@gmail.com>.
+
 2021-09-13  Patrice Dumas  <pertusus@free.fr>
 
 	* tp/Texinfo/Convert/LaTeX.pm: protect [ and ] with braces as they
diff --git a/tp/Texinfo/XS/parsetexi/end_line.c b/tp/Texinfo/XS/parsetexi/end_line.c
index a5bd0ce..c973cc1 100644
--- a/tp/Texinfo/XS/parsetexi/end_line.c
+++ b/tp/Texinfo/XS/parsetexi/end_line.c
@@ -778,8 +778,8 @@ kdbinputstyle_invalid:
     case CM_headings:
       {
         if (!strcmp (line, "off") || !strcmp (line, "on")
-            || !strcmp (line, "double") || !strcmp (line, "singleafter")
-            || !strcmp (line, "doubleafter"))
+            || !strcmp (line, "single") || !strcmp (line, "double")
+            || !strcmp (line, "singleafter") || !strcmp (line, "doubleafter"))
           {
             ADD_ARG(line);
           }
-- 
cgit v1.1

